test_fms/fms/test_unstructured_fms_io.F90

```diff
@@ -48,9 +48,7 @@ program test_unstructured_fms_io
                                             fms_io_exit
     implicit none
 
-#ifdef use_netCDF
 #include <netcdf.inc>
-#endif
```

Comment:
If this is the only place the macro exists in this file, I don't know that we even need the #include

Reply:
I'll make sure this line is/is not needed, and delete if not.  Thank you.

Reply:
Not needed.  New version pushed.